test/swar/BasicOperations.cpp

```diff
@@ -0,0 +1,19 @@
+#include "zoo/swar/metaLog.h"
+#include "zoo/swar/SWAR.h"
+
+#include "catch2/catch.hpp"
+
+#include <type_traits>
+
+using namespace zoo;
+
+TEST_CASE(
+    "Basic SWAR",
+    "[swar]"
+) {
+    uint8_t v = 0x8;
+    SECTION("Simple Mask") {
+      auto mask = swar::makeBitmask<8, uint32_t>(v);
+      CHECK(mask == 0x08080808ull);
+    }
+}
```

Comment:
Test the things taht can be tested at compilation time at compilation time.  As of now, there's no need to include catch here either